corpusreaders/src/main/java/edu/illinois/cs/cogcomp/nlp/corpusreaders/ontonotes/viewgenerators/NERViewGenerator.java

```diff
@@ -0,0 +1,177 @@
+package edu.illinois.cs.cogcomp.nlp.corpusreaders.ontonotes.viewgenerators;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import edu.illinois.cs.cogcomp.core.datastructures.IntPair;
+import edu.illinois.cs.cogcomp.core.datastructures.ViewNames;
+import edu.illinois.cs.cogcomp.core.datastructures.textannotation.Constituent;
+import edu.illinois.cs.cogcomp.core.datastructures.textannotation.Sentence;
+import edu.illinois.cs.cogcomp.core.datastructures.textannotation.SpanLabelView;
+import edu.illinois.cs.cogcomp.core.datastructures.textannotation.TextAnnotation;
+
+/**
+ * This class will read NER data from the OntoNotes 5.0 corpora stored in it's SQL representation 
+ * in a mySQL database and produce a TextAnnotation containing the sentences, tokens and NER view.
+ * @author redman
+ * @deprecated we have decided NOT to use the database version of the OntoNotes 5.0 corpora, just too shoddy.
+ */
```

Comment:
ideally, use annotation AvoidUsing: https://github.com/CogComp/cogcomp-nlp/blob/master/core-utilities/src/main/java/edu/illinois/cs/cogcomp/core/utilities/AvoidUsing.java
-- allows you to specify what class to use instead.